tests/e2e_logging.py

```diff
@@ -434,6 +439,41 @@ def test_historical_query(network, args):
     return network
 
 
+@reqs.description("Read historical receipts")
+@reqs.supports_methods("log/private", "log/private/historical_receipt")
+def test_historical_receipts(network, args):
```

Comment:
Could you add this to the test suite as well?

Reply:
Yes!